index.js

```diff
@@ -3,7 +3,16 @@
  * @param {object} assignments - An object with assignments organized by category.
  * @returns {string[]} An array of strings, which represent all of the keys in the object.
  */
-function getAssignmentTypes(assignments) {}
+function getAssignmentTypes(assignments) {
+  let keys = Object.keys(assignments)
+ 
+  for(let _categories in assignments) {
+console.log(Object.keys(assignments))
+  }
+ 
+  return keys
```

Comment:
Well done! A couple notes:

1. Please remove `console.log` statements prior to submission.
2. You don't _need_ the `keys` variable. Though you're _certainly_ welcome to prefer breaking it down in the end, try making this all one line for an excellent exercise in concise coding.